src/VisualStudio/Core/Def/NavigateTo/RoslynSearchItemsSourceProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.LanguageServices;
+using Microsoft.VisualStudio.Search.Data;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.NavigateTo
+{
+    /// <summary>
+    /// Roslyn implementation of the <see cref="ISearchItemsSourceProvider"/>.  This is the entry-point from VS to
```

Comment:
should we delete https://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/NavigateTo/VisualStudioNavigateToItemProviderFactory.cs,16 ?